src/main/java/uk/ac/ebi/eva/pipeline/configuration/io/writers/StoreSSIdWriterConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
```

Comment:
Update Copyright years across the newly created files...

Reply:
done